internal/agent/device/spec/spec.go

```diff
@@ -78,7 +80,7 @@ func (s *Manager) GetRendered(ctx context.Context) (v1alpha1.RenderedDeviceSpec,
 		return v1alpha1.RenderedDeviceSpec{}, v1alpha1.RenderedDeviceSpec{}, false, err
 	}
 
-	desired, skipSync, err := s.getDesiredRenderedSpec(ctx, current.Owner, current.TemplateVersion)
+	desired, skipSync, err := s.getDesiredRenderedSpec(ctx, current.Owner, EmptyTemplateVersion)
```

Comment:
If I have the current and its empty during bootstrap then it should return the latest templateVersion? Then moving forward we continue each time we will pass the last known template version. So it would be OK?